src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -268,10 +247,6 @@ private function createRawDomDocument(string $html): void
     /**
      * Returns the HTML with added document type, Content-Type meta tag, and self-closing slashes, if needed,
      * ensuring that the HTML will be good for creating a DOM document from it.
-     *
-     * @param string $html
-     *
-     * @return string the unified HTML
```

Comment:
"Unified" was not an appropriate description here (it means 'made into a unit'), so I agree it's best to simply remove the `@return` annotation entirely.